src/styles/sidebar/components/group-list.scss

```diff
@@ -61,10 +114,63 @@ $group-list-spacing-below: 50px;
     margin-right: 2px;
   }
 
+  .group__icon--unavailable {
+    fill: $gray-light;
+    float: right;
+    height: 20px;
+    width: auto;
+  }
+
   .group-details {
     flex-grow: 1;
     flex-shrink: 1;
   }
+  .group-details-community-groups {
+    flex-grow: 1;
+    flex-shrink: 1;
+    font-weight: 500;
+
+    .group-details__unavailable-message,
+    .group-details__actions {
+      display: none;
+    }
+
+    &.expanded {
+      .group-details__toggle {
+        display: none;
+      }
+
+      .group-details__unavailable-message,
+      .group-details__actions {
+        display: block;
+      }
+    }
+  }
+
+  .group-details__toggle {
+    font-size: 0.8em;
```

Comment:
I know that Dave Woolfe had differing opinions on this, but we generally avoid em-sizing in our current CSS and prefer specifying pixel sizes directly. em-sizes are a scaling of the font-size in the parent. The problem is that if you nest elements that define em-sizes they compound (eg. if you had an element with font-size 0.8em and a child also with font-size 0.8em then the actual font size in the child will be 0.8 * 0.8 * {inherited font size in parent}).

The final font size here is 12px, which happens to match the `$body1-font-size`, so we can just replace 0.8em with that.